api/src/schemas/email-subscription/resubscribe.ts

```diff
@@ -0,0 +1,20 @@
+import { Type } from '@fastify/type-provider-typebox';
+
+export const resubscribe = {
+  params: Type.Object({
+    unsubscribeId: Type.String({
+      minLength: 1
+    })
+  }),
+  response: {
+    302: Type.Object({
+      // this doesn't seem to work
+      headers: Type.Object({
+        location: Type.String()
+      })
+    }),
```

Comment:
I do not see this being used in the old api. So, does it need to work?

Reply:
no